summary-report.py

```diff
@@ -37,16 +37,51 @@
 # User editable variables END
 ##########
 
+#To the arguments handling
+import sys
+import getopt
+
 import socket
 from pygtail import Pygtail
 import smtplib
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
+try: 
+  opts, args = getopt.getopt(sys.argv[1:],"hh:p:f:t:s:b:",["help="])
+except getopt.GetoptError:
+  print 'Try to use the help with the followign command: summary-report.py -h'
+  sys.exit(2)
+
+for opt,arg in opts:
+  if opt == '-h':
+    print """summary-report.py has the following arguments options:
+      -p <production>
+      -f <sender email>
+      -t <recipients emails>
+      -s <smtp domain>
+      -b <base url>
+    """
+    sys.exit()
+  elif opt in ("-p"):
```

Comment:
Since you're using single flags per if statement, wouldn't it be better to use
```elif opt == "-p"```
and so on?